data/default_release_images.json

```
@@ -146,5 +146,36 @@
         ],
         "url": "quay.io/openshift-release-dev/ocp-release:4.14.1-multi",
         "version": "4.14.1-multi"
-    }
+    },
+    {
+        "openshift_version": "4.15",
+        "cpu_architecture": "x86_64",
+        "cpu_architectures": [
+          "x86_64"
+        ],
+        "url": "quay.io/openshift-release-dev/ocp-release:4.15.0-ec.1-x86_64",
+        "support_level": "beta",
+        "version": "4.15.0-ec.1"
+      },
+      {
+        "openshift_version": "4.15",
+        "cpu_architecture": "arm64",
+        "cpu_architectures": [
+          "arm64"
+        ],
+        "url": "quay.io/openshift-release-dev/ocp-release:4.15.0-ec.1-aarch64",
+        "support_level": "beta",
+        "version": "4.15.0-ec.1"
+      },
+      {
+        "openshift_version": "4.15-multi",
+        "cpu_architecture": "multi",
+        "cpu_architectures": [
+          "x86_64",
+          "arm64"
+        ],
```

```suggestion
        "cpu_architectures": [
          "x86_64",
          "arm64"
          "ppc64le",
          "s390x"
        ],
```
to align with 4.14/4.13 config